src/widget/datetime/datetimepicker-extended.js

```diff
@@ -121,8 +122,17 @@ class DatetimepickerExtended extends Widget {
 
     update() {
         const $dateTimeI = $( this.element );
-        const val = ( $dateTimeI.val().length > 0 ) ? new Date( $dateTimeI.val() ).toISOLocalString() : '';
-
+        let val = ( $dateTimeI.val().length > 0 ) ? new Date( $dateTimeI.val() ).toISOLocalString() : '';
+        /**
+         * fix a bug which is only on safari (#745)
+         * If the local timezone is +08:00, for a date value of new Date('2020-10-10T13:10:10') will be:
+         * *** in chrome, the value is: Sat Oct 10 2020 13:10:10 GMT+0800 (China Standard Time)
+         * *** but in safari, the value is: Sat Oct 10 2020 21:10:10 GMT+0800 (CST)
+         * so we have to append the timezone here
+         */
+        if ( os.safari ) {
+            val = ( $dateTimeI.val().length > 0 ) ? ( new Date( $dateTimeI.val() + new Date( dt ).timezoneOffsetAsTime ) ).toISOLocalString() : '';
```

Comment:
`dt` is not defined (see also Travis error in PR)

Reply:
Thanks, fixed now.